File: test/support/models/log_city_location.js
'use strict';

var mongoose = global.mongoose || require('mongoose');

module.exports = mongoose.model('LogCityLocation', new mongoose.Schema({
  _id: String,
  country: String,
  region: String,
  city: String,
  location: mongoose.Schema.Types.Mixed,
  updated_at: Date,
}, { collection: 'log_city_locations' }));

File: test/ruby/support/models/log_city_location.rb
class LogCityLocation
  include Mongoid::Document
  field :_id, :type => String, :overwrite => true, :default => lambda { SecureRandom.uuid }
  field :country, :type => String
  field :region, :type => String
  field :city, :type => String
  field :location, :type => Hash
  field :updated_at, :type => Time
end
